code/modules/urist/structures/boardingstructures.dm

```diff
@@ -30,13 +30,17 @@
 /obj/structure/boarding/shipportal/proc/teleport(atom/movable/M as mob|obj)
 	if(istype(M, /obj/effect)) //sparks don't teleport
 		return
-	else
-		var/tele_x = GLOB.using_map.overmap_ship.evac_x
-		var/tele_y = GLOB.using_map.overmap_ship.evac_y
-		var/tele_z = GLOB.using_map.overmap_ship.evac_z
+	if(istype(M, /mob/living))
```

Comment:
make this controlled by a var so we can change this if we want to for events

Reply:
Didn't think of that! Boolean `block_pirate_teleport` has been added!